mesonbuild/interpreter/interpreter.py

```diff
@@ -2353,6 +2353,14 @@ def install_data_impl(self, sources: T.List[mesonlib.File], install_dir: str,
     def func_install_subdir(self, node: mparser.BaseNode, args: T.Tuple[str],
                             kwargs: 'kwargs.FuncInstallSubdir') -> build.InstallDir:
         exclude = (set(kwargs['exclude_files']), set(kwargs['exclude_directories']))
+
+        srcdir = os.path.join(self.environment.source_dir, self.subdir, args[0])
+        mlog.log(list(os.walk(srcdir)))
```

Comment:
I think that's leftover for debugging?

Reply:
Oops, yeah. Debugging stuff blends in when you add a single big block of code, apparently.

Is this otherwise good?